start_pokealarm.py

```diff
@@ -133,7 +133,7 @@ def parse_settings(root_path):
                         help='Specify either metric or imperial units to use for distance measurements. ')
     parser.add_argument('-ct', '--cache_type', type=parse_unicode, action='append', default=['mem'],
                         choices=cache_options,
-                        help='Specify a Google API Key to use.')
+                        help='Select cache storage location (mem or file)')
```

Comment:
Change this to `Specify the type of cache to use. Options: ['mem', 'file'] (Default: 'mem')`